lib/frontend/src/Obelisk/Frontend.hs

```diff
@@ -67,11 +67,14 @@ type ObeliskWidget t route m =
   , MonadFix (Performable m)
   , PrimMonad m
   , Prerender t m
-  , SetRoute t route m
-  , SetRoute t route (Client m)
+  , PrebuildAgonstic t route m
+  , PrebuildAgonstic t route (Client m)
+  )
+
+type PrebuildAgonstic t route m =
```

Comment:
“Agnostic” ?

Reply:
Constraints that apply in the prebuilding and regular building case, and a non-exported name anyways.

Reply:
Yeah that makes sense. I'm just pointing out the typo